projects/mci/src/mci_output.c

```diff
@@ -91,6 +109,7 @@ StatusCode mci_output_init(MotorControllerOutputStorage *storage, Mcp2515Storage
   };
   storage->motor_can = motor_can;
   status_ok_or_return(pedal_rx_init(&storage->pedal_storage, &pedal_settings));
+  // status_ok_or_return(regen_braking_init());
```

Comment:
We shouldn't leave commented-out code lying around, it's not gonna be needed in the future.

Reply:
[done](url)